torchbenchmark/models/alexnet/__init__.py

```diff
@@ -8,13 +8,13 @@
 
 class Model(BenchmarkModel):
     task = COMPUTER_VISION.CLASSIFICATION
-    def __init__(self, device="cpu", jit=False):
+    def __init__(self, device=None, jit=False):
         super().__init__()
         self.device = device
         self.jit = jit
         self.model = models.alexnet().to(self.device)
         if self.jit:
-            self.model = torch.jit.script(self.model)
+            self.model = torch.jit.optimize_for_inference(torch.jit.script(self.model))
```

Comment:
self.model is also used for train() and get_module(), could you please create another model instance and name it jit_freeze_model? And in eval(), use self.jit_freeze_model to evaluate. For `get_modules()`, I prefer to use the train model, so no need to change it.